src/py42/clients/_archiveaccess/__init__.py

```diff
@@ -0,0 +1,165 @@
+import posixpath
+from collections import namedtuple
+
+from py42.exceptions import Py42ArchiveFileNotFoundError
+
+
+FileSelection = namedtuple(u"FileSelection", u"file, num_files, num_dirs, num_bytes")
+
+
+class FileType(object):
+    DIRECTORY = u"DIRECTORY"
+    FILE = u"FILE"
+
+
+class ArchiveAccessor(object):
```

Comment:
Can we get docstrings on all these classes to make it easier to see how they all fit together?